docs/templates/eap-cd-amq-persistent-s2i.adoc

```diff
@@ -243,18 +244,19 @@ for more information.
 |`JGROUPS_ENCRYPT_PASSWORD` | The password for the keystore and certificate | `${JGROUPS_ENCRYPT_PASSWORD}`
 |`JGROUPS_CLUSTER_PASSWORD` | JGroups cluster password | `${JGROUPS_CLUSTER_PASSWORD}`
 |`AUTO_DEPLOY_EXPLODED` | Controls whether exploded deployment content should be automatically deployed | `${AUTO_DEPLOY_EXPLODED}`
-.11+| `${APPLICATION_NAME}-amq`
+.12+| `${APPLICATION_NAME}-amq`
 |`AMQ_USER` | User name for standard broker user. It is required for connecting to the broker. If left empty, it will be generated. | `${MQ_USERNAME}`
 |`AMQ_PASSWORD` | Password for standard broker user. It is required for connecting to the broker. If left empty, it will be generated. | `${MQ_PASSWORD}`
+|`AMQ_ROLE` | AMQ Role for authenticated user | `${MQ_ROLE}`
 |`AMQ_TRANSPORTS` | -- | `${MQ_PROTOCOL}`
-|`AMQ_QUEUES` | Queue names, separated by commas. These queues will be automatically created when the broker starts. Also, they will be made accessible as JNDI resources in EAP. | `${MQ_QUEUES}`
-|`AMQ_TOPICS` | Topic names, separated by commas. These topics will be automatically created when the broker starts. Also, they will be made accessible as JNDI resources in EAP. | `${MQ_TOPICS}`
+|`AMQ_QUEUES` | Queue names, separated by commas. These queues will be automatically created when the broker starts. Also, they will be made accessible as JNDI resources in EAP. Note that all queues used by the application *must* be specified here in order to be created automatically on the remote AMQ broker. | `${MQ_QUEUES}`
+|`AMQ_ADDRESSES` | -- | `${MQ_TOPICS}`
```

Comment:
AMQ_ADDRESSES ? What is it? Shouldn't be AMQ_TOPICS?

Reply:
This is a new AMQ 7 thing, addresses abstract both queues and topics. I've tried to make the templates combine both queues and topics into addresses where relevant, but I'll recheck this one.

Reply:
@tremes See here: https://github.com/jboss-container-images/jboss-amq-7-broker-openshift-image/blob/amq71-dev/modules/amq-launch/added/launch.sh#L30